### PR TITLE
Fix Buster

### LINKED ISSUE #5: [buster] Install fails because of some PHP error ?
Relevant bits from the instal log : 

```
5%: Starting installation and creating directories.
10%: Creating database tables.
Unable to install database: "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?
File:
/var/www/concrete5/concrete/src/Package/StartingPointPackage.php:384
```

### PR DESCRIPTION
**Pull request by yalh76**
Fix #5 

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/concrete5_ynh%20PR6%20(yalh76)/)

### CODE REVIEW

**Comment by maniackcrudelis** [comment]
With the version 8.5.2, install seems ok on package_check on Buster. (Although still running...)
I also did a real install on Buster, it seems to work, I went to the admin panel.

**Comment by maniackcrudelis** [comment]
Package_check ok with your last commit on Buster.

**Comment by yalh76** [comment]
Well for now, package check is ok on buster but installation fails on stretch with the error: 
```
Database should support the configured character set and collation... The database does not have a complete support for all the Unicode characters. This implies, for example, that you may not be able to save emoticons (it's the same behavior as concrete5 versions older than 8.5.0). You should upgrade your database to fix this issue.
```

**Comment by arthurlutz** [comment]
Fix seems to work on my test instance, thanks for the fixes !

**Comment by yalh76** [comment]
> 
> 
> Fix seems to work on my test instance, thanks for the fixes !

and you are on Buster or Stretch ?

**Comment by arthurlutz** [comment]
on buster

**Comment by yalh76** [comment]
> 
> 
> on buster

Cool, the fix works for Buster but not on Stretch... 

So maybe when YunoHost 4.0 / Buster will be released, we can merge that PR...

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # concrete5_ynh
```
Comment (ericgaspar) [inline README.md:1]:
```suggestion
# Concrete5 for YunoHost
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**